virttest/libvirt_xml/vm_xml.py

```diff
@@ -1472,6 +1472,42 @@ def remove_all_boots(self):
             pass  # Element already doesn't exist
         self.xmltreefile.write()
 
+    def set_boot_order_by_target_dev(self, target, order):
+        """
+        Set boot order by target dev
+
+        :param target: The target dev on host
+        :param order: The boot order number
+        """
+        devices = self.get_devices()
+        for device in devices:
+            if device.device_tag == "disk":
+                if device.target.get("dev") == target:
+                    device.boot = order
+        self.set_devices(devices)
+
+    def set_os_attrs(self, **attr_dict):
+        """
+        Set attributes of VMOSXML
+
+        :param attr_dict: The key words of os attributes
+        """
+        try:
+            os_xml = getattr(self, "os")
```

Comment:
I think it is better to use like os_xml = self.os instead of getattr to make it easy to understand

Reply:
kyla, it will cause inspekt lint failed

Reply:
[root@localhost virttest]# inspekt lint libvirt_xml/vm_xml.py
Pylint disabled: W,R,C,E1002,E1101,E1103,E1120,F0401,I0011
Pylint enabled : W0611
************* Module avocado-vt.virttest.libvirt_xml.vm_xml
E0203:1497,21: VMXML.set_os_attrs: Access to member 'os' before its definition line 1502
Lint: libvirt_xml/vm_xml.py FAIL 
Syntax check FAIL
[root@localhost virttest]#

Reply:
Change the operation to another func can resolve the syntax check issues, anyway, it is also a method to get attr, so it is ok.